src/test/java/teammates/test/pageobjects/InstructorFeedbackResultsPage.java

```diff
@@ -138,6 +138,26 @@ public void clickShowStats() {
 
     public void clickIndicateMissingResponses() {
         click(indicateMissingResponsesCheckbox);
+        expandPanels();
+    }
+
+    public void expandPanels() {
+        try {
+            // In case the panels are already expanded
```

Comment:
Based on this PR, panels should not be already expanded, am I right? In that case the test should fail if the panels are already expanded?